.github/dependabot.yml

```diff
@@ -28,6 +28,28 @@ updates:
     pull-request-branch-name:
       separator: "-" 
 
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+    labels:
+      - "dependencies"
+    groups:
+      all-dependencies:
+        patterns:
+          - "*"
+    commit-message:
+      prefix: "[NPM] "
+    reviewers:
+      - "matomo-org/core-reviewers"
+    pull-request-branch-name:
+      separator: "-"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-minor"]
```

Comment:
Note: This will currently prevent any minor updates for npm library to happen. At some point we might want to remove that, but at the moment this would break the build process. Applying only patch version updates doesn't cause problems in the build process.